tests/infrastructure/prometheus.go

```diff
@@ -260,17 +231,15 @@ var _ = DescribeInfra("[rfe_id:3187][crit:medium][vendor:cnv-qe@redhat.com][leve
 		// but if the default disk is not vda, the test will break
 		// TODO: introspect the VMI and get the device name of this
 		// block device?
-		vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-		tests.AppendEmptyDisk(vmi, "testdisk", v1.VirtIO, "1Gi")
-
+		vmi := libvmi.NewAlpine(libvmi.WithEmptyDisk("testdisk", v1.VirtIO, resource.MustParse("1G")))
 		if preferredNodeName != "" {
-			pinVMIOnNode(vmi, preferredNodeName)
-		}
-		nodeName := startVMI(vmi)
-		if preferredNodeName != "" {
-			Expect(nodeName).To(Equal(preferredNodeName), "Should run VMIs on the same node")
+			vmi = libvmi.NewAlpine(libvmi.WithEmptyDisk("testdisk", v1.VirtIO, resource.MustParse("1G")),
```

Comment:
nit: what about constructing a slice of options and then passing it to `NewAlpine`?